app/experimenter/nimbus-ui/package.json

```diff
@@ -21,7 +21,8 @@
     "collectCoverageFrom": [
       "src/**/*",
       "!src/**/*.stories.*",
-      "!src/lib/{mocks,test-utils}.tsx"
+      "!src/**/mocks.*",
```

Comment:
Found that I was able to share some mocks / utils between stories & tests for this component, so I figured I'd just make the coverage exception more general